frontend/components/RepoCardItem/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Heading, HEADING_OPTIONS } from '../shared/Heading';
+import Button from '../shared/Button';
+import githubLogo from '../../public/static/images/github.png';
+import './style.css';
+
+const RepoCardItem = ({ subTitle, cardDesc, isHoverd }) => {
```

Comment:
- It is a good practice to give it a generic  name it ( Card) .it may be used for ( Projects, batches, ) or